components/climate/climate_ir.rst

```diff
@@ -247,6 +247,42 @@ Known working with:
 
 - Delonghi PAC WE 120HP
 
+.. _mitsubishi:
+
+``mitsubishi`` Climate
+------------------------
+
+Additonal configurations available for this platform.
+
+Configuration variables:
+
+- **set_fan_speeds** (*Optional*): Used to set non-default fan speeds. When not used default is **fan_low** = ``fan1``, **fan_medium** = ``fan2``, **fan_hi** = ``fan3``
+
+  - **fan_low** (*Required*, string)
+  - **fan_medium** (*Required*, string)
+  - **fan_hi** (*Required*, string)
+  - Options are ``fan1``, ``fan2``, ``fan3``, ``fan4``, ``fan5``
+- **horizontal_default** (*Optional*, string): What to default to when the AC unit's horizontal direction is *not* set to swing. Default is ``middle``. Options are: ``left``, ``middle-left``, ``middle``, ``middle-right``, ``right``, ``auto``
+- **vertical_default** (*Optional*, string): What to default to when the AC unit's vertical direction is *not* set to swing. Default is ``middle``. Options are: ``down``, ``mdown``, ``middle``, ``mup``, ``up``, ``auto``
```

Comment:
As per the changes to be made in code PR
```suggestion
- **vertical_default** (*Optional*, string): What to default to when the AC unit's vertical direction is *not* set to swing. Default is ``middle``. Options are: ``down``, ``middle-down``, ``middle``, ``middle-up``, ``up``, ``auto``
```

Reply:
done